unittest/test_Util.cpp

```diff
@@ -658,8 +668,12 @@ TEST_CASE("Util::read_file and Util::write_file")
   CHECK_THROWS_WITH(Util::read_file("does/not/exist"),
                     "No such file or directory");
 
+#ifdef _WIN32
+  CHECK_THROWS_WITH(Util::write_file("", "does/not/exist"), "Invalid argument");
```

Comment:
Can the error message be fixed? It's rather unmatching to the error case.

Reply:
Apparently the test was writing to a file named empty string, when writing to an actual non-existent path, the error is what you would expect. I can't even blame Windows for this, and I blame Windows for a lot of things.

I added a check for empty path, and added a test for a non-empty non-existent path.